docs/src/main/asciidoc/deploying-to-openshift.adoc

```diff
@@ -75,6 +75,8 @@ Building is handled by the link:container-image#s2i[container-image-s2i] extensi
 
 The build that will be performed is an s2i binary build. The input of the build is the jar that has been built locally and the output of the build is an ImageStream that is configured to automatically trigger a deployment.
 
+NOTE: During the build you may find the `Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed` exception due to self-signed certificate. To solve this, just add the following line to your `application.properties`: `quarkus.kubernetes-client.trust-certs=true`. For more information, see link:https://quarkus.io/guides/deploying-to-kubernetes.[deploying to kubernetes]
```

Comment:
It would be nice to give more emphasis on the code that needs to be copy/pasted, like:

```suggestion
NOTE: During the build you may find the `Caused by: javax.net.ssl.SSLHandshakeException: sun.security.validator.ValidatorException: PKIX path building failed` exception due to self-signed certificate. To solve this, just add the following line to your `application.properties`: 

```properties
quarkus.kubernetes-client.trust-certs=true
```

For more information, see link:https://quarkus.io/guides/deploying-to-kubernetes.[deploying to kubernetes]
```

Reply:
@gastaldi do I need to amend/merge the commits? Or is it ok to have 2 commits in a pull request?

Reply:
In this case since they are directly related (the second commit fixes the first one) it's better to squash both commits. 
You can have multiple commits if they complement the PR (eg. 1 commit for changes in module A and another for module B, etc)

Reply:
@gastaldi can you check if it's ok now?